desktop_version/src/main.cpp

```diff
@@ -380,6 +369,7 @@ int main(int argc, char *argv[])
     char* langDir = NULL;
     char* fontsDir = NULL;
     bool seed_use_sdl_getticks = false;
+    bool editor_disabled = !BUTTONGLYPHS_keyboard_is_available();
```

Comment:
Note for me: Make sure this checks the SteamDeck environment variable, we can maybe adjust this to keyboard_is_preferred since consoles very very quietly support kb/m 🤫

Reply:
Oh I didn't see this before committing -- I added it to `keyboard_is_available`, is that okay?

Reply:
lgtm!

Reply:
> we can maybe adjust this to keyboard_is_preferred since consoles very very quietly support kb/m 🤫

I guess the conditional in `BUTTONGLYPHS_keyboard_is_available()` could always be changed to also return true if keyboard_is_active, even if SteamDeck?

Reply:
Something along those lines - the general guideline is to start by assuming controller input, and if keyboard/mouse events occur then we can assume it is explicitly requested by the user.